force-app/recipes/classes/ApexMockeryOverview.cls

```diff
@@ -12,27 +12,36 @@ private class ApexMockeryOverview {
     Mock deliveryServiceMock = Mock.forType(DeliveryService.class);
     MethodSpy planDeliverySpy = deliveryServiceMock.spyOn('planDelivery');
     Bakery myBakery = new Bakery((DeliveryService) deliveryServiceMock.stub);
-    planDeliverySpy.whenCalledWith(new Pastry('Chocolatine')).thenReturn(Date.today().addDays(3));
+    planDeliverySpy.whenCalledWith(new Pastry('Chocolatine')).thenReturn(Date.today().addDays(3)).thenReturnOnce(Date.today().addDays(2));
```

Comment:
question: couldn't it be useful to have `thenReutnr(...., times(1))` notation or something similar? So that we can have more that one call stubbed

Reply:
Yep ! With the actual implementation we could easily add an API `thenReturnsTime(final Integer times, final Object value);` 
I can add it here, or we could do that in another PR, what do you think ?